utils/statediff/src/main.rs

```diff
@@ -46,11 +46,8 @@ fn open_db(chain_path: &Path) -> forest_db::rocks::RocksDb {
 #[cfg(feature = "paritydb")]
 fn open_db(chain_path: &Path) -> forest_db::parity_db::ParityDb {
```

Comment:
Can't this code use the shared `open_db` function?

Reply:
Can't because of cyclic dependency. I'm wondering if there's a reason statediff is separate, rather than within cli crate as a subcommand?